pages/index.tsx

```diff
@@ -1,4 +1,6 @@
 import Head from "next/head";
+import NextScript from "next/script";
+
```

Comment:
```suggestion
```

Please remove this import statement if we're not using it anywhere within this file.